test/test_openapi.py

```diff
@@ -120,6 +121,31 @@ def route2():
                 pass
         self.assertEqual(str(cm.exception), 'Only specify path_spec once per route path')
 
+    def test_shared_path_spec(self):
+        """
+        Assert that, when sharing the path_spec, routes don't overwrite each
+        other's properties.
+        """
+        app = AzulChaliceApp('testing', spec={'foo': 'bar'})
+        shared_path_spec = {
+            'parameters': [
+                params.query('foo', schema.optional({'type': 'string'})),
+            ]
+        }
+        for i in range(2):
+            @app.route(
+                f'/swagger-test-{i}',
```

Comment:
```suggestion
            @app.route(f'/swagger-test-{i}',
```

Aligned indent until the lines get too long.

Reply:
Sorry about that... I confused myself when I read "wrap all"